jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -73,12 +97,29 @@ pipeline {
             agent { label agent_nodes["x64"] }
             steps {
                 script {
-                    if (VERSION == '') {
+                    def url_paths = [:]
+
+                    OPENSEARCH_ARTIFACT_URL ? url_paths["opensearch"] = OPENSEARCH_ARTIFACT_URL : null
+                    OPENSEARCH_DASHBOARDS_ARTIFACT_URL ? url_paths['opensearch-dashboards'] = OPENSEARCH_DASHBOARDS_ARTIFACT_URL : null
+
+                    if (OPENSEARCH_DASHBOARDS_ARTIFACT_URL && OPENSEARCH_ARTIFACT_URL == "") {
                         currentBuild.result = 'ABORTED'
-                        error('VERSION cannot be empty. Please provide version of the artifacts to validate')
+                        error("Provide OPENSEARCH_ARTIFACT_URL to validate")
+                    }
+
+                    if(OPENSEARCH_ARTIFACT_URL) {
+                        ARCHITECTURE = url_paths.values().find { it.contains("x64") } ? "x64" : "arm64"
```

Comment:
This is not true I believe? Windows does not have arm64.

Reply:
I see @gaiksaya currently we do not support windows. But that is based on user input. When windows arm64 does not exist we do not provide the artifacts details at all while calling the job . It should always be x64 when it comes to windows.

Reply:
Do we need to add check for that? I beleive we need to keep this in mind when we add validation for windows?

Reply:
Yeah Sayali. I will add a check when we add windows.